packages/fxa-content-server/tests/server/routes/get-config.js

```diff
@@ -1,60 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-const { registerSuite } = intern.getInterface('object');
-const assert = intern.getPlugin('chai').assert;
-const sinon = require('sinon');
-const route = require('../../../server/lib/routes/get-config');
-var instance;
-var request;
-var response;
-
-registerSuite('routes/get-config', {
-  'route interface is correct': function () {
-    assert.isFunction(route);
-    assert.lengthOf(route, 0);
-  },
-
-  'initialise route': {
-    before: function () {
-      instance = route();
-    },
-
-    tests: {
-      'instance interface is correct': function () {
-        assert.isObject(instance);
-        assert.lengthOf(Object.keys(instance), 3);
-        assert.equal(instance.method, 'get');
-        assert.equal(instance.path, '/config');
```

Comment:
So some of these routes are definately still being used in our frontend. This test calls the `/config` endpoint which settings uses.

It seems strange to me to remove the test but keep the api route functionality. The team mentioned that these tests are not being run. Would it make sense to migrate them off of intern instead?

To my understanding, when we move completely to a React frontend, some of these backend routes will still be used.

Also, if a test exists but does not run...does it really exists 😅